src/Symfony/Component/Notifier/Bridge/Engagespot/EngagespotTransport.php

```diff
@@ -27,15 +27,15 @@
  */
 final class EngagespotTransport extends AbstractTransport
 {
-    protected const HOST = 'api.engagespot.co/2/campaigns';
+    protected const HOST = 'api.engagespot.co/v3/notifications';
 
     private $apiKey;
-    private $campaignName;
+    private $apiSecret;
 
-    public function __construct(string $apiKey, string $campaignName, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    public function __construct(string $apiKey, string $apiSecret, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

Comment:
Replacing the second argument is a BC break (and a silent one here as it will start using the campaign name as api secret)

Reply:
please use `#[\SensitiveParameter]`

Reply:
Hi, I'm the founder & maintainer of Engagespot. I can confirm that nobody is using version v2 of Engagespot Symfony integration. So isn't it is safe to do this change there is no concern of backward compatibility?

Reply:
Packagist: https://packagist.org/packages/symfony/engagespot-notifier/stats
@anandrmedia What if someone starts using the integration via Symfony 6.0 and then move on to 6.1?
In any case, this PR is close as there was no more activity. Someone would need to take over and create a new one to move things forward.